src/components/thumbnailTherapy/TherapyButton/TherapyButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from "react";
+
+interface IButton {
+  text: string;
+}
+
+const TherapyButton: FC<IButton> = ({ text }) => {
+  return (
+    <button className="rounded-lg bg-transparent w-full h-10 font-black font-lato text-black hover:text-white hover:bg-black text-black font-black">
+      {text}
+    </button>
+  );
+};
+
+export default TherapyButton;
```

Comment:
we don't need this subfolders, we need only thumbnailTherapy folder, or just name this folder thumbnail, so under this folder it is ok to just name this component Button. It will looks like src/component/thumbnail/Button.tsx, also other compents needs refactor